.github/workflows/branch-snapshot.yml

```diff
@@ -28,15 +28,13 @@ jobs:
           go-version: '1.21.4'
       - name: Install Nats Server
         run: |
+          pkill -9 nats-server 2>/dev/null || true
+          mkdir -p ~/.local/bin
           cd $GITHUB_WORKSPACE
           git clone https://github.com/nats-io/nats-server.git
           cd nats-server
-          go get
-          go build main.go
-          mkdir -p ~/.local/bin
-          cp main ~/.local/bin/nats-server
-          cd ..
-          rm -rf nats-server
+          git checkout tags/v2.11.8 -b release
```

Comment:
Right now I need to run against 2.11.8 instead of head of main. Really need to figure out how to just get the latest release. The scripts in use elsewhere are unreliable.